projects/demo/src/pages/recipes/card/examples/1-basic/component.ts

```diff
@@ -0,0 +1,31 @@
+import {ChangeDetectionStrategy, Component} from '@angular/core';
+import {MaskitoOptions} from '@maskito/core';
+import {maskitoDateOptionsGenerator} from '@maskito/kit';
+
+@Component({
+    selector: 'card-doc-example-1',
+    templateUrl: 'template.html',
+    styleUrls: ['style.less'],
```

Comment:
```suggestion
    styleUrls: ['./style.less'],
```

it is better to use in this way because i remember case (in proprietary demo) that it can cause bugs sometimes